File: Models/Forum.cs
using System.Collections.Generic;

namespace WebForums.Models
{
	public class Forum
	{
		public int ID { get; set; }
		public virtual Category Category { get; set; }
		public string Title { get; set; }
		public string Description { get; set; }
		public virtual ICollection<Thread> Threads { get; set; }
	}
}

File: src/entities/Forum.ts
import { Entity, PrimaryGeneratedColumn, Column, ManyToOne, OneToMany } from "typeorm"
import { Category } from "./Category";
import { Thread } from "./Thread";

@Entity()
export class Forum {
    @PrimaryGeneratedColumn()
    id: number;

    @ManyToOne(() => Category, category => category.forums)
    category: Category;

    @Column()
    title: string;

    @Column()
    description: string;

    @OneToMany(() => Thread, thread => thread.forum)
    threads: Thread[];
}